src/main/java/apoc/metrics/Metrics.java

```diff
@@ -0,0 +1,111 @@
+package apoc.metrics;
+
+import apoc.ApocConfiguration;
+import apoc.load.LoadCsv;
+import apoc.util.FileUtils;
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+import java.io.File;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class Metrics {
+    /**
+     * DAO for a single line in a metrics/*.csv file.
+     * Value is abstracted to double; in reality some are int, some are float, some are double.
+     */
+    public class Neo4jMetricEntry {
+        public long t;
+        public double value;
+        public String metric;
+
+        public Neo4jMetricEntry(long t, double value, String metric) {
+            this.t = t;
+            this.value = value;
+            this.metric = metric;
+        }
+    }
+
+    public class Neo4jMetric {
+        public String name;
+        public long lastUpdated;
+        public String path;
```

Comment:
should we expose the path to the outside world?

Reply:
I thought it might be useful to some but it's not a hard requirement so safer to leave out, it can always be added later.

Reply:
Ok, then let's leave it out for now.